development/sentry-upload-artifacts.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+readonly __SCRIPT_NAME__="${0##*/}"
+readonly __SEE_HELP_MESSAGE__="See '${__SCRIPT_NAME__} --help' for more information."
+
+function die {
+  local message="${1}"
+
+  printf "ERROR: %s\\n" "${message}" >&2
```

Comment:
Nit: I believe if we use single quotes here we don't need to escape the backslash

```suggestion
  printf 'ERROR: %s\n' "${message}" >&2
```